lib/Redis.php

```diff
@@ -14,6 +14,34 @@
  * @package		Resque/Redis
  * @author		Chris Boulton <chris@bigcommerce.com>
  * @license		http://www.opensource.org/licenses/mit-license.php
+ *
+ * @mixin Credis_Cluster
+ * @method string get(string $key)
+ * @method bool exists(string $key)
+ * @method array smembers(string $set)
+ * @method bool sismember(string $set, string $key)
+ * @method bool del(string $key)
+ * @method void set(string $key, mixed $value)
+ * @method bool expire(string $key, int $time)
+ * @method bool ping(?string $value = null)
+ * @method bool incrby(string $key, int $increment)
+ * @method bool decrby(string $key, int $decrement)
+ * @method bool rpush(string $key, string ...$element)
+ * @method array blpop(string|array $key, int $timeout)
+ * @method string|bool lpop(string $key, int $count = 1)
+ * @method string|bool rpop(string $key, int $count = 1)
+ * @method int rpoplpush(string $source, string $destination)
+ * @method int|false llen(string $key)
+ * @method array lrange(string $key, int $start, int $stop)
+ * @method void zadd(string $key, int $score, mixed ...$value)
+ * @method int zcard(string $key)
+ * @method int sadd(string $key, string $member)
+ * @method int srem(string $key, string $member)
+ * @method int lrem(string $key, int $count, string $element)
+ * @method array keys(string $pattern)
+ * @method int zrem(string $key, string|int ...$member)
+ * @method array zrangebyscore(string $key, int|string $min, int|string $max, array $options = [])
```

Comment:
I have a list sourced from Credis that I put in #94. I think that one is slightly more accurate, considering Credis is the main supported driver